app/views/partners/children/show.html.erb

```diff
@@ -59,7 +59,11 @@
                     <dd><%= @child.health_insurance %></dd>
 
                     <dt>Item needed:</dt>
-                    <dd><%= current_partner.organization.item_id_to_display_string_map[@child.item_needed_diaperid] %></dd>
```

Comment:
Note: I'm not entirely sure of the data model, but there is potentially a data migration issue here. 

When this code is live, you'll have the `items` association but what happens to this `item_needed_diaperid` value that existed on records prior.